pull_request_template.md

```diff
@@ -1,6 +1,6 @@
 ## Description
 
-___Add your PR description here, deleting this example___
+Update product doc repository to include searching case data
 
```

Comment:
Why are we changing this? Isn't the line you're trying to replace in the template specific to what you did for this pull request? I get it if this was part of the searchandfilter.md file but not in the generic template

Reply:
I believe that was mistake, I'll reach out to Dionis and get it figured out. Good catch though, we don't want to mess with the original template file.